flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/debezium/reader/SnapshotSplitReader.java

```diff
@@ -86,6 +87,8 @@ public class SnapshotSplitReader implements DebeziumReader<SourceRecord, MySqlSp
     public AtomicBoolean hasNextElement;
     public AtomicBoolean reachEnd;
 
+    private static final long EXECUTOR_AWAIT_TERMINATION_TIMEOUT_SECONDS = 5;
```

Comment:
```suggestion
    private static final long READER_CLOSE_TIMEOUT = 30L;
```
And let's rename `executor` to `executorService` for more precise meaning.